docs/uperf.md

```diff
@@ -35,6 +35,34 @@ Once done creating/editing the resource file, you can run it by:
 # kubectl apply -f <path_to_file> # if created a new cr file
 ```
 
+Note: If you'd like to try to experiment with storing results in a pv and have followed
+instructions to deploy operator with attached pvc and would like to send results to ES.
+you can instead define a custom resource like
+```yaml
+apiVersion: benchmark.example.com/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  es:
+    server: <es_host>
+    port: <es_port>
+    index: <es_index>
+  user: rht_perf_ci
+  store_results: true
+  results:
```

Comment:
I wonder if we could store this in the defaults, and allow the user to override (which should never be the case).

Reply:
You mean `results.path` should be set in `group_vars/all.yaml` and then for it to be overwritten through CR? but what about the inconsistency that's bound to be produced if the user doesn't mount it on the same locally for operator pod? I'd say it leave it as is, as it doesn't seem like we'd be able to solve a problem but if anything for a person developing on ripsaw it adds to more confusion